CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 This file is intended to be modified using the [`changelog`](https://github.com/cucumber/changelog) command-line tool.
 
 ## [Unreleased]
+- Stop printing a warning about `config.cache_classes` being set to `false`
```

Comment:
Because of the way our changelog parser / automation works these all need a subheading. This should be `### Removed` (See in the v4 / v3 releases how they're structured.